tastyworks/models/trading_account.py

```diff
@@ -129,7 +206,7 @@ def from_dict(cls, data: dict) -> List:
         async with aiohttp.request('GET', url, headers=session.get_request_headers()) as response:
             if response.status != 200:
                 raise Exception('Could not get open positions info from Tastyworks...')
-            data = (await response.json())['data']['items']
+            data = Position.list_from_dict((await response.json())['data']['items'])
```

Comment:
Could you please update the return type comment (line 199) for this function as it now returns a List of positions rather than a dict